app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DisplayValue from '../../../../../components/FormComponents/DisplayValue'
+
+const InformationItem = ({ label, value, showIfMissing }) => {
+  if (!showIfMissing && !value) {
+    return null
+  }
+
+  return (
+    <div className="row border-bottom-line">
+      <div className="col-md-6 col-xs-6">
+        <span> {label} </span>
+      </div>
+      <div className="col-md-6 col-xs-6">
+        <strong>
+          {' '}
+          <DisplayValue value={value} />{' '}
```

Comment:
I reckon we could put the `<strong>` inside `<DisplayValue />`

Reply:
No longer using DisplayValue